src/admin/migrations/m180703_183052_add_category_alias.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m180703_183052_add_category_alias
+ */
+class m180703_183052_add_category_alias extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn('news_cat', 'alias', $this->text());
```

Comment:
Maybe we could rename to slug? I think the slug field should be varchar `$this->string()` and can not be null `->notNull()`

Reply:
I just don't like the sound of a word `slug` so it's never my first choice, but English is my, like, third, language, so if you think slug is a better choice - let's go with it.

Reply:
we used alias too, but slug is more analog to yii framework: https://www.yiiframework.com/doc/api/2.0/yii-helpers-baseinflector#slug()-detail